examples/debugging/simple_spread/feedforward/decentralised/run_ippo.py

```diff
@@ -104,7 +104,7 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         run_evaluator=True,
         sample_batch_size=5,
         num_epochs=15,
-        num_executors=1,
```

Comment:
I think we should keep the example at 1 executor for now.

Reply:
Oh yes I made it just for testing, will do that